artview/components/plot_radar.py

```diff
@@ -44,6 +44,7 @@ class RadarDisplay(Component):
     Vgatefilter = None  #: see :ref:`shared_variable`
     VplotAxes = None  #: see :ref:`shared_variable` (no internal use)
     VpathInteriorFunc = None  #: see :ref:`shared_variable` (no internal use)
+    VpyartDisplay = None  #: see :ref:`shared_variable`
```

Comment:
I continue to be agains classes accessing each other and I do like how the concept of shared variables forces the class to organize it self. I don't think this is something I will be changing in the future. That being said, I do see some space for increase interaction and coordination between the components. Modes are good, but they are still not perfect and have some performance issues.